tools/distrib/check_nanopb_output.sh

```diff
@@ -55,3 +56,25 @@ if ! diff -r "$NANOPB_TMP_OUTPUT" src/core/ext/filters/client_channel/lb_policy/
   echo "Outputs differ: $NANOPB_TMP_OUTPUT vs $LOAD_BALANCER_GRPC_OUTPUT_PATH"
   exit 2
 fi
+
+#
+# Checks for handshaker.proto and transport_security_common.proto
```

Comment:
What about altscontext.proto?

Reply:
The changes are applied. Previously, altxcontext.proto is not compiled at all since it is not yet used by ALTS C stack. But I think we can still pre-generate its compiled output and make it part of ALTS core just in case we may need it in future.